AOT compilation: can't resolve module ./src/toast.component

* **I'm submitting a ...**
[x ] bug report
[ ] feature request
[ ] question about the decisions made in the repository

* **Do you want to request a *feature* or report a *bug*?**
Not sure if it is a bug but I have an error when executing AOT compilation.

* **What is the current behavior?**
When I try to AOT compile my project where we included ng2-toasty, I obtain the following error:
```
[07:05:01]  ngc started ... 
can't resolve module ./src/toast.component from /var/lib/jenkins/jobs/project/workspace/src/node_modules/ng2-toasty/index.d.ts
[07:05:03]  build prod failed: Cannot read property 'ToastComponent' of undefined, resolving symbol ToastyModule in 
            /var/lib/jenkins/jobs/project/workspace/src/node_modules/ng2-toasty/index.d.ts, resolving 
            symbol ToastyModule in 
            /var/lib/jenkins/jobs/project/workspace/src/node_modules/ng2-toasty/index.d.ts 
[07:05:03]  ionic-app-script task: "build" 
[07:05:03]  Error: Cannot read property 'ToastComponent' of undefined, resolving symbol ToastyModule in 
            /var/lib/jenkins/jobs/project/workspace/src/node_modules/ng2-toasty/index.d.ts, resolving 
            symbol ToastyModule in 
            /var/lib/jenkins/jobs/project/workspace/src/node_modules/ng2-toasty/index.d.ts 
Error: Cannot read property 'ToastComponent' of undefined, resolving symbol ToastyModule in /var/lib/jenkins/jobs/project/workspace/src/node_modules/ng2-toasty/index.d.ts, resolving symbol ToastyModule in /var/lib/jenkins/jobs/project/workspace/src/node_modules/ng2-toasty/index.d.ts
    at simplifyInContext (/var/lib/jenkins/jobs/project/workspace/src/node_modules/@angular/compiler-cli/src/static_reflector.js:475:23)
    at StaticReflector.simplify (/var/lib/jenkins/jobs/project/workspace/src/node_modules/@angular/compiler-cli/src/static_reflector.js:478:22)
    at StaticReflector.annotations (/var/lib/jenkins/jobs/project/workspace/src/node_modules/@angular/compiler-cli/src/static_reflector.js:60:36)
    at NgModuleResolver.resolve (/var/lib/jenkins/jobs/project/workspace/src/node_modules/@angular/compiler/bundles/compiler.umd.js:14261:46)
    at CompileMetadataResolver._loadNgModuleMetadata (/var/lib/jenkins/jobs/project/workspace/src/node_modules/@angular/compiler/bundles/compiler.umd.js:14646:45)
    at CompileMetadataResolver.getUnloadedNgModuleMetadata (/var/lib/jenkins/jobs/project/workspace/src/node_modules/@angular/compiler/bundles/compiler.umd.js:14636:23)
    at addNgModule (/var/lib/jenkins/jobs/project/workspace/src/node_modules/@angular/compiler/bundles/compiler.umd.js:12944:43)
    at /var/lib/jenkins/jobs/project/src/node_modules/@angular/compiler/bundles/compiler.umd.js:12957:16
    at Array.forEach (native)
    at _createNgModules (/var/lib/jenkins/jobs/project/workspace/src/node_modules/@angular/compiler/bundles/compiler.umd.js:12956:28)

npm ERR! Linux 3.14.32-xxxx-grs-ipv6-64
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "run" "ionic:build" "--prod"
npm ERR! node v7.4.0
npm ERR! npm  v4.0.5
```

* **Please tell us about your environment:**

- Ionic 2
- Angular version: 2.2.1
- node v7.4.0
- npm  v4.0.5

* **Other information** 
In index.ts, we have 2 exports:
```
export * from './src/toasty.service';
export * from './src/toasty.component';
```

Adding to those exports an export for toast.component seems to fix the compilation error.

--- a/index.ts
+++ b/index.ts
@@ -7,6 +7,7 @@ import { CommonModule } from '@angular/common';
 
 export * from './src/toasty.service';
 export * from './src/toasty.component';
+export * from './src/toast.component';
 
 import { ToastyComponent } from './src/toasty.component';
 import { ToastComponent } from './src/toast.component';

--- a/index.ts
+++ b/index.ts
@@ -8,6 +8,7 @@ import { CommonModule } from '@angular/common';
 export * from './src/toasty.service';
 export * from './src/toasty.component';
 export * from './src/toast.component';
+export * from './src/shared';
 
 import { ToastyComponent } from './src/toasty.component';
 import { ToastComponent } from './src/toast.component';

